packages/create-block/lib/init-block.js

```diff
@@ -38,8 +39,9 @@ async function initBlockJSON( {
 	info( 'Creating a "block.json" file.' );
 
 	const outputFile = plugin
-		? join( process.cwd(), slug, folderName, 'block.json' )
-		: join( process.cwd(), slug, 'block.json' );
+		? join( process.cwd(), targetDir, slug, folderName, 'block.json' )
+		: join( process.cwd(), folderName, slug, 'block.json' );
```

Comment:
From https://github.com/WordPress/gutenberg/pull/53781#issuecomment-1841144219:

> I just tested this and noticed that if you combine it with the --no-plugin argument, it seems to ignore the --target-dir argument.

It looks like `targetDir` should be used here as well.

Reply:
I've changed how the paths are done so this is no longer relevant.